quadrat/functions.php

```diff
@@ -50,6 +50,27 @@ function quadrat_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'quadrat_scripts' );
 
+/**
+ * Add a filter to the render callback of the footer, 
+ * so it can be translated or customized for WordPress.com.
+ */
+function quadrat_add_markup_to_footer_template( $block_content, $block ) {
+	if ( $block['blockName'] === 'core/template-part' && $block['attrs']['slug'] === 'footer' ) {
+		// If we are on WPCOM, do not render a credit so that the WP.com footer credit plugin can handle it
+		$wp_credit = '';
+		if ( ! class_exists( 'WPCOM_Block_Theme_Footer_Credits') ){
```

Comment:
Since this is environment-specific does it make sense to put it into the wpcom-specific code that isn't included in the build?